ta/sims/ta_sims.c

```diff
@@ -212,3 +212,32 @@ TEE_Result sims_entry_panic(void *session_context, uint32_t param_types,
 
 	return TEE_SUCCESS;
 }
+
+TEE_Result sims_entry_open_session(void **session,
+				   uint32_t param_types, TEE_Param params[4])
+{
+	struct sims_session *loc_ctx = NULL;
+	TEE_Result res = TEE_ERROR_GENERIC;
+	uint32_t loc_ptypes = 0;
+	TEE_Param loc_params[4] = { };
+
+	res = open_client_session(&loc_ctx);
+	if (res != TEE_SUCCESS)
+		return res;
+
+	if (param_types ==
```

Comment:
If it is not GitHub fooling me here, then the indentation line 228-231 makes it pretty hard to see what is what.

Reply:
:)